clients/src/main/java/org/apache/kafka/common/requests/OffsetCommitRequest.java

```diff
@@ -89,33 +88,29 @@ public Map<TopicPartition, Long> offsets() {
         return offsets;
     }
 
-    public static List<OffsetCommitResponseTopic> getErrorResponseTopics(
-            List<OffsetCommitRequestTopic> requestTopics,
```

Comment:
Did we always just pass in all the topics here? I see the usage below that was removed, but curious if we would ever want to make a response for just some topics.

Reply:
The usage below was the single usage of this method so we always used it with all topics. For the context, I refactored it because I use the refactored one in one of the next patch.